packages/integration-tests/test/challengeExitTest.js

```diff
@@ -45,88 +46,157 @@ describe('Challenge exit tests', async () => {
     ;[aliceAccount, bobAccount] = await helper.createAndFundManyAccounts(web3, accounts[0], '', [INTIIAL_ALICE_AMOUNT, INTIIAL_BOB_AMOUNT])
```

Comment:
Weeeeeelll, you see in Javascript you don't have to use semicolons to end a line, except for when you do. 

In certain cases (if you start a line with [, (,  +, *, /, -, ,, .) then you really should have a semicolon on the previous statement, otherwise javascript thinks that it's a continuation of that statement. But of course the `standard` code standard doesn't let you put semicolons on the end of a line, so the brilliant and stylish solution is to put it at the start of the next line.

Now, you might ask why I need to start a line with [  because after all, that [will never appear at the start of a line in 'real' code](https://standardjs.com/rules.html#semicolons) buuuuut in ES6 javascript, [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) is now the _preferred_ way to unpack arrays and objects, so starting a line with [ will be quite common.

Oh, what a wondrous thing is Javascript 😬